### ISSUE TITLE
Have Cluster and NodeGroup return version

### ISSUE THREAD

**Issue by cdenneen** [crossplane-contrib/provider-aws#1184]
Currently right now if you describe the cluster or nodegroup the version returns from API but from CR it is not listed.
I think having this return will help in providing detail to the underlying resources. 
Also `additionalPrinterColumns` should be provided to display the data that returns from atProvider like platformVersion and after this is added the k8s version for the control plane as well as the nodegroup. (addons as well).

**Comment by cdenneen** [comment]
Currently cluster `atProvider` returns:
```
status:
  atProvider:
    arn: arn:aws:eks:us-east-1:XXX:cluster/test-eks-2-t8rxp
    certificateAuthorityData: XXX
    createdAt: "2022-03-11T18:44:38Z"
    endpoint: https://XXX.gr7.us-east-1.eks.amazonaws.com
    identity:
      oidc:
        issuer: https://oidc.eks.us-east-1.amazonaws.com/id/XXX
    platformVersion: eks.3
    resourcesVpcConfig:
      clusterSecurityGroupId: sg-XXX
      vpcId: vpc-XXX
    status: UPDATING
```
and nodeGroup:
```
status:
  atProvider:
    createdAt: "2022-03-11T19:03:08Z"
    modifiedAt: "2022-03-11T19:06:05Z"
    nodeGroupArn: arn:aws:eks:us-east-1:XXX:nodegroup/test-eks-2-t8rxp/test-eks-2-rt4st/10bfbd2e-7be6-5d92-9ac4-06fb85f49bf6
    nodeGroupHealth: {}
    resources:
      autoScalingGroup:
      - name: eks-10bfbd2e-7be6-5d92-9ac4-06fb85f49bf6
    scalingConfig:
      desiredSize: 2
    status: ACTIVE
```

the API will return version:

```json
{
    "cluster": {
        "name": "test-eks-2-t8rxp",
        "arn": "arn:aws:eks:us-east-1:XXX:cluster/test-eks-2-t8rxp",
        "createdAt": "2022-03-11T13:44:38.369000-05:00",
        "version": "1.20",
        "endpoint": "https://XXX.gr7.us-east-1.eks.amazonaws.com",
        "roleArn": "arn:aws:iam::XXX:role/test-eks-2-4fvwh",
        "resourcesVpcConfig": {
            "subnetIds": [
                "subnet-XXX",
                "subnet-YYYY"
            ],
            "securityGroupIds": [
                "sg-XXXX"
            ],
            "clusterSecurityGroupId": "sg-XXXX",
            "vpcId": "vpc-XXXX",
            "endpointPublicAccess": false,
            "endpointPrivateAccess": true,
            "publicAccessCidrs": []
        },
        "kubernetesNetworkConfig": {
            "serviceIpv4Cidr": "172.20.0.0/16",
            "ipFamily": "ipv4"
        },
        "logging": {
            "clusterLogging": [
                {
                    "types": [
                        "api",
                        "audit",
                        "authenticator",
                        "controllerManager",
                        "scheduler"
                    ],
                    "enabled": false
                }
            ]
        },
        "identity": {
            "oidc": {
                "issuer": "https://oidc.eks.us-east-1.amazonaws.com/id/XXXX"
            }
        },
        "status": "UPDATING",
        "certificateAuthority": {
            "data": "XXXX"
        },
        "platformVersion": "eks.3",
        "tags": {
            "crossplane-providerconfig": "apss",
            "crossplane-kind": "cluster.eks.aws.crossplane.io",
            "crossplane-name": "test-eks-2-t8rxp"
        }
    }
}
```
nodegroup
```json
{
    "nodegroup": {
        "nodegroupName": "test-eks-2-rt4st",
        "nodegroupArn": "arn:aws:eks:us-east-1:XXXX:nodegroup/test-eks-2-t8rxp/test-eks-2-rt4st/10bfbd2e-7be6-5d92-9ac4-06fb85f49bf6",
        "clusterName": "test-eks-2-t8rxp",
        "version": "1.20",
        "releaseVersion": "1.20.11-20220309",
        "createdAt": "2022-03-11T14:03:08.370000-05:00",
        "modifiedAt": "2022-03-11T14:24:19.805000-05:00",
        "status": "ACTIVE",
        "capacityType": "ON_DEMAND",
        "scalingConfig": {
            "minSize": 1,
            "maxSize": 100,
            "desiredSize": 2
        },
        "instanceTypes": [
            "t3.small"
        ],
        "subnets": [
            "subnet-XXXX",
            "subnet-YYYY"
        ],
        "amiType": "AL2_x86_64",
        "nodeRole": "arn:aws:iam::XXX:role/test-eks-2-vb2fs",
        "labels": {},
        "resources": {
            "autoScalingGroups": [
                {
                    "name": "eks-10bfbd2e-7be6-5d92-9ac4-06fb85f49bf6"
                }
            ]
        },
        "diskSize": 20,
        "health": {
            "issues": []
        },
        "updateConfig": {
            "maxUnavailable": 1
        },
        "tags": {
            "crossplane-providerconfig": "apss",
            "crossplane-kind": "nodegroup.eks.aws.crossplane.io",
            "crossplane-name": "test-eks-2-rt4st"
        }
    }
}
```

So preferrably the provider-aws will populate cluster version and nodegroup version and releaseVersion

**Comment by haarchri** [comment]
fixed with #1204
